rosidl_runtime_cpp/include/rosidl_runtime_cpp/traits.hpp

```diff
@@ -15,11 +15,138 @@
 #ifndef ROSIDL_RUNTIME_CPP__TRAITS_HPP_
 #define ROSIDL_RUNTIME_CPP__TRAITS_HPP_
 
+#include <codecvt>
+#include <string>
 #include <type_traits>
 
 namespace rosidl_generator_traits
 {
 
+inline void value_to_yaml(bool value, std::ostream & out)
+{
+  out << (value ? "true" : "false");
+}
+
+inline void character_value_to_yaml(unsigned char value, std::ostream & out)
+{
+  auto flags = out.flags();
+  out << "0x" << std::hex << std::setw(2) << std::setfill('0') << \
```

Comment:
This causes a downstream build failure in `rmw_dds_common`

https://ci.ros2.org/job/ci_linux/12266/console

```
--- stderr: rmw_dds_common
15:49:21 In file included from /home/jenkins-agent/workspace/ci_linux/ws/build/rmw_dds_common/rosidl_generator_cpp/rmw_dds_common/msg/detail/gid__traits.hpp:9,
15:49:21                  from /home/jenkins-agent/workspace/ci_linux/ws/build/rmw_dds_common/rosidl_generator_cpp/rmw_dds_common/msg/gid.hpp:9,
15:49:21                  from /home/jenkins-agent/workspace/ci_linux/ws/src/ros2/rmw_dds_common/rmw_dds_common/include/rmw_dds_common/gid_utils.hpp:21,
15:49:21                  from /home/jenkins-agent/workspace/ci_linux/ws/src/ros2/rmw_dds_common/rmw_dds_common/src/gid_utils.cpp:21:
15:49:21 /home/jenkins-agent/workspace/ci_linux/ws/install/rosidl_runtime_cpp/include/rosidl_runtime_cpp/traits.hpp: In function ‘void rosidl_generator_traits::character_value_to_yaml(unsigned char, std::ostream&)’:
15:49:21 /home/jenkins-agent/workspace/ci_linux/ws/install/rosidl_runtime_cpp/include/rosidl_runtime_cpp/traits.hpp:33:35: error: ‘setw’ is not a member of ‘std’
15:49:21    33 |   out << "0x" << std::hex << std::setw(2) << std::setfill('0') << \
15:49:21       |                                   ^~~~
15:49:21 /home/jenkins-agent/workspace/ci_linux/ws/install/rosidl_runtime_cpp/include/rosidl_runtime_cpp/traits.hpp:33:51: error: ‘setfill’ is not a member of ‘std’; did you mean ‘fill’?
```

Reply:
Possibly just needs the `<iomanip>` include https://en.cppreference.com/w/cpp/io/manip/setw